src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -334,5 +334,8 @@ The first part date {1} should be lesser than or equal to the second part date {
   <data name="ValueLengthIsNotRequiredLength" xml:space="preserve">
     <value>Value length is not {0}.</value>
     <comment>{0} required length.</comment>
-  </data>  
+  </data>
+  <data name="CustomTagsAlreadyChanged" xml:space="preserve">
+    <value>One or more custom tags have been changed.</value>
```

Comment:
Exception messages should be understandable by customers and actionable. I am not able to understand what you are trying to convey here.